GeoFire.podspec

```diff
@@ -1,11 +1,11 @@
 Pod::Spec.new do |s|
```

Comment:
Does it require 2.2?  If it doesn't then what's the harm of leaving it for more compatibility?